### DISCUSSION TITLE
Documentation how to switch to dark mode

### DISCUSSION THREAD

**Discussion by marbx** [i3/i3#5627] [Q&A] [answered]
One reason using i3 is its excellent documentation and the "Only documented behavior is supported" principle.

[Screenshots and videos](https://i3wm.org/screenshots/) have both light and dark mode.
Right after install, `$mod+Enter` opens a light background terminal, and applications have light colored borders.
The documentation does not mention dark mode
[This article](https://askubuntu.com/questions/1,275273/how-do-i-change-my-color-theme-for-i3) suggests to replace the window manager, which I cannot believe is necessary.

How do you switch i3, GTK and Qt applications to dark mode?

**Comment by okraits** [comment]
`$mod+Enter` looks for installed terminals and executes the first one it finds. The colors used by the terminal are the terminal's default colors, i3 has nothing to do with them.

i3 doesn't have a dark mode, but you can easily change its colors: https://i3wm.org/docs/userguide.html#client_colors

Changing the GTK or Qt theme is beyond i3's scope. For GTK, you can use a tool like `lxappearance`.

**Comment by marbx** [comment]
Thanks, I actually like the colors of  i3status and I was unaware of lxappearance.

I would have appreciated this in the User Guide:

Regarding "dark mode": i3 has nothing to do with the colors of applications, and you can changes the theme of them using:
- `lxappearance` for GTK applications like Firefox or GNOME Terminal
- `qt5ct`  for QT5 applications like Krusader

 Shall I open a PR to the User Guide?

**Reply by okraits** [reply]
I think that it doesn't make sense to add this to the User Guide because:
* Similar to this, we could add hints on how to change the screen resolution/setup, change the audio volume, change network settings etc. I think this is out of the scope of i3, the window manager itself.
* These hints can become outdated and need to be maintained.

In contrast to a desktop environment which includes all needed tools to execute the mentioned tasks, a standalone window manager is often chosen by people who are willing to do the research on how to achieve the mentioned tasks. A good resource is the Arch Linux wiki: https://wiki.archlinux.org/

As an alternative, i3 can be used to replace the default window manager of a desktop environment so that all the tools of the desktop environment can still be used.

**Reply by orestisfl** [reply, marked as answer]
> Shall I open a PR to the User Guide?

Even though it's nice to have more information in the user guide, elaborating on details that i3 has nothing to do with is a good way to end up with outdated and misleading information. For example, `qt5ct` is applicable for Qt 5 while Qt 4 had `qtconfig-qt4`: https://wiki.archlinux.org/title/Qt#Configuration_of_Qt_5_applications_under_environments_other_than_KDE_Plasma.

I would only mention that i3 is not responsible for the styling of applications and that you can configure that either through GTK and Qt-wide options or through the application directly.
